ansible/create-gh-deployment.yaml

```diff
@@ -9,7 +9,6 @@
       new_image_tag: "{% if 'prod' in gh_env %}{{ prod_image_tag }}{% elif 'test' in gh_env %}{{ test_image_tag }}{% elif 'dev' in gh_env %}{{ dev_image_tag }}{% endif %}"
       suffix: "{{ new_image_tag }}"
       version: "{{ new_image_tag }}"
-      ref: refs/pull/{{ PR }}/head
```

Comment:
its used to target a github deployment and update its status to pending. Prior we'd implicitly set the ref to be the PR branch, but instead we should be targeting the head ref of the PR instead of an implicit branch.